pkg/adapter/awssqs/adapter.go

```diff
@@ -82,43 +87,36 @@ func (a *Adapter) pollLoop(ctx context.Context, q *sqs.SQS, stopCh <-chan struct
 	logger := logging.FromContext(ctx)
 	cctx, cancel := context.WithCancel(ctx)
```

Comment:
Not needed anymore.

Reply:
https://github.com/knative/eventing-sources/pull/124/commits/573ca8b6814d24cd42ef3c01f50a7ec7bd76106b